cmd/keymasterd/config.go

```diff
@@ -201,11 +202,27 @@ type DenyKeyConfig struct {
 	KeyDenyFPsshSha256 []string `yaml:"key_deny_list_ssh_sha256"`
 }
 
+type ExternalSignerType int
+
+const (
+	ExternalSignerInvalid ExternalSignerType = iota
+	ExternalSignerYubiPIV
+	ExternalSignerAWSKMS
```

Comment:
Please change to `ExternalSignerAWS`. See note below.

Reply:
I am fine with making the external facing interace just AWS, but I think having the interals different will make other logic simpler as you can just look at the enum to determiina action, otherwise you will also need another reparsing and switch based on the arn type. Maybe make these internal (lowercase)

Reply:
Sure.